docs/UserGuide.md

```diff
@@ -115,17 +166,33 @@ Format: `list -a`
 - Give at least one space between list and -a.
 </box>
 
-### Clearing all suppliers and deliveries : `clear`
+[Back to Table of Contents](#table-of-contents)
+
+---
+
+### 3.2.3 Clearing all suppliers and deliveries : `clear`
 
 Clear all data regarding suppliers and deliveries in vendor vault.
```

Comment:
I think this can be changed to "Clears all data..." to standardise with the other commands